### PR TITLE
Change "window core" group

### PR DESCRIPTION
**Pull request by terrakok**
Rename the window core group "org.jetbrains.window.core" -> "org.jetbrains.androidx.window"

### CODE REVIEW

File: libraryversions.toml [hunk 1]
```diff
@@ -292,7 +292,7 @@ WEAR_TILES = { group = "androidx.wear.tiles", atomicGroupVersion = "versions.WEA
 WEAR_WATCHFACE = { group = "androidx.wear.watchface", atomicGroupVersion = "versions.WEAR_WATCHFACE" }
 WEBKIT = { group = "androidx.webkit", atomicGroupVersion = "versions.WEBKIT" }
 WINDOW = { group = "androidx.window", atomicGroupVersion = "versions.WINDOW" }
-WINDOW_CORE = { group = "org.jetbrains.window.core", atomicGroupVersion = "versions.WINDOW", overrideInclude = [ ":window:window-core" ]  }
+WINDOW_CORE = { group = "org.jetbrains.androidx.window.core", atomicGroupVersion = "versions.WINDOW", overrideInclude = [ ":window:window-core" ]  }
```
Comment (igordmn) [inline libraryversions.toml]:
```suggestion
WINDOW_CORE = { group = "org.jetbrains.androidx.window", atomicGroupVersion = "versions.WINDOW", overrideInclude = [ ":window:window-core" ]  }
```

Comment (igordmn) [reply libraryversions.toml]: core is the name of the module, not of the group

Comment (igordmn) [reply libraryversions.toml]: Also, to be consistent with other libraries, we should move it after `SAVEDSTATE_JETBRAINS` and rename to `WINDOW_CORE_JETBRAINS`

Comment (terrakok) [reply libraryversions.toml]: agree, but then why are the next lines:
```
WINDOW_EXTENSIONS = { group = "androidx.window.extensions", atomicGroupVersion = "versions.WINDOW_EXTENSIONS" }
WINDOW_EXTENSIONS_CORE = { group = "androidx.window.extensions.core", atomicGroupVersion = "versions.WINDOW_EXTENSIONS_CORE" }
```

Comment (igordmn) [reply libraryversions.toml]: We shouldn't look at them. We should look at [the group](https://mvnrepository.com/artifact/androidx.window/window-core) of published Androidx artifacts. Our approach is to replace "androidx" by "org.jetbrains.androidx"

**Review by igordmn — APPROVED** [review]

**Pull request merged**